packages/node_modules/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -422,32 +421,18 @@ skipInNode(describe)('plugin-meetings', () => {
             assert.equal(alice.meeting.mediaProperties.shareTrack.getConstraints().height, 720);
           }
           assert.equal(alice.meeting.isSharing, true);
+          assert.equal(alice.meeting.shareStatus, 'local_share_active');
+          assert.equal(bob.meeting.shareStatus, 'remote_share_active');
           console.log('SCREEN SHARE PARTICIPANTS ', JSON.stringify(alice.meeting.locusInfo.participants));
 
           return testUtils.waitUntil(10000);
         }));
 
-      it('alice stops sharing ', () => Promise.all([
```

Comment:
I've removed this test, because we already do "bob stops sharing" later on so the scenario of stopping sharing was duplicated. Instead I've changed it now so that Bob steals the share from Alice - that's something that wasn't tested at all before